src/components/Books/Books.js

```diff
@@ -1,27 +1,25 @@
 import React from "react";
-import BookM, { deleteBook } from "../../redux/books/Book";
+import { useSelector } from "react-redux";
 import Book from "./Book";
 
 const Books = () => {
-  const normal = BookM("G");
+  const bookQuantity = useSelector((state) => state.books.length);
+  const books = useSelector((state) => state.books);
   return (
-    <ul className="booklist">
-      {" "}
-      {normal.books.map((book) => {
-        const { id, title, author, genre } = book;
-        return (
-          <li key={book.id}>
-            <Book
-              id={id}
-              title={title}
-              genre={genre}
-              author={author}
-              onClick={deleteBook}
-            />{" "}
-          </li>
-        );
-      })}{" "}
-    </ul>
+    <div>
+      <h2>Number of books {bookQuantity} </h2>{" "}
+      <ul className="booklist">
+        {" "}
+        {books.map((book) => {
+          const { id, title, author, genre } = book;
+          return (
+            <li key={book.id}>
+              <Book id={id} title={title} genre={genre} author={author} />{" "}
+            </li>
+          );
+        })}{" "}
+      </ul>{" "}
+    </div>
   );
 };
```

Comment:
- [ ] When this component mounts, a call to the API should be made to retrieve all of the books available already in the API. Remember that this must be an action in the redux store and it should be dispatched when this component finished mounting.